pandas/core/arrays/string_.py

```diff
@@ -73,51 +81,134 @@ class StringDtype(ExtensionDtype):
     Examples
     --------
     >>> pd.StringDtype()
-    StringDtype
+    string[python]
     """
 
-    name = "string"
-
     #: StringDtype.na_value uses pandas.NA
     na_value = libmissing.NA
+    _metadata = ("storage",)
+
+    def __init__(self, storage=None):
```

Comment:
Can you update the docstring above for this new keyword?

Reply:
done in d9dcd20f99ca6fc2f0a08eb2a3f4a54f29309af3